jkube-kit/config/resource/src/main/java/org/eclipse/jkube/kit/config/resource/RuntimeMode.java

```diff
@@ -33,36 +33,23 @@
      * onto cluster. It can be used both on vanilla Kubernetes and
      * OpenShift.
      */
-    kubernetes(false, "Kubernetes"),
+    KUBERNETES("Kubernetes"),
```

Comment:
Plexus deserialization is not configurable for enums (this is the reason why this values were still in lower-case).
Make sure that this value is never deserialized from a Maven property (i.e. `jkube.mode`), if it still is deserialized somewhere, it will fail).